observer.py

```diff
@@ -34,6 +34,7 @@ def get_publishers(network):
 
 
 async def main(args):
+    symbols_to_ignore = set(["Crypto.ORCA/USD"])
     program_key = get_key(network=args.network, type="program", version="v2")
```

Comment:
would be nice if the symbols were configurable via args. I believe these are command line args, which would be useful for turning off symbols on the fly.